tests/cypress/support/selectors.js

```diff
@@ -10,4 +10,10 @@ export default {
     exportViewPopover: "[data-test='export-view-popover']",
     logoImage: "[data-test='logo-image']",
     loader: "[data-test='loader']",
+    videoGuide: "[data-test=video-guide-button]",
+    timelineButton: "app-timeline > .ion-no-padding",
```

Comment:
Please avoid using general selectors as they may change during development.

Add a test specific tag attribute like `[data-test='timeline-button']` as in the other selector tags. [Here is the Cypress documentation for reference.](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)

Reply:
for video guide able to find general selectors but for lead time don't able to find as you mentioned in comment.
Thats the reason i have used relative xpath which help in dynamic pages.